integration-tests/failing-noop.spec.ts

```diff
@@ -74,44 +73,5 @@ CONFIGS().forEach(({ lib, rpc, setup, protocol }) => {
       done();
     });
 
-    it('Verify that the local forger can forge the failing_noop operation on the genesis block', async (done) => {
```

Comment:
we still need to test forger, can you add a test case in `integration-tests/data/allTestsCases.ts`